Dockerfile.django-alpine

```diff
@@ -16,13 +16,15 @@ RUN \
     bind-tools \
     mysql-client \
     mariadb-dev \
-    postgresql14-client \
+    postgresql16-client \
```

Comment:
What's the thinking on rev'ing this to postgresql16? Some distros (RHEL 8 comes to mind) don't have support for Postgres less then 13 - which is still supported by the upstream project till Nov 2025 (see [here](https://www.postgresql.org/support/versioning/)).

While I'm running PostgreSQL 16, not all of the broader DefectDojo community has the option to run that version.

Reply:
hi @mtesauro ,
in the default django docker image which is based in debian we're installing the postgres-client in version 15. the defectdojo helm chart postgres database dependency uses version 15. since the postgres client is backward compatible with older versions (see the [Notes section](https://www.postgresql.org/docs/current/app-psql.html)) there should be no issue using a older postgres database instance.

Reply:
OK. That works for me - I just don't want to optimize for one distro and break others if we don't absolutely have to.